### ISSUE TITLE
`state.get(i)` returns the value at `i`, not the record at `i`

### ISSUE THREAD

**Issue by cowboyd** [flexyford/impagination#15]
If you have a record with the value "bob", you expect `state.get(0)` to return a record like:
```javascript
{
  offset: 0
  isPending: false,
  isRejected false,
  isSettled: true,
  content: "bob"
}
```

Instead you see `"bob"`

**Comment by flexyford** [comment]
If that's the case then we can forget about returning `state.pages` since that's what we used to compute the page numbers in the `color-demo`. We can easily get the page numbers by using the `Math.floor(record.offset / pageSize)`

**Comment by flexyford** [comment]
Fixed on #18
